### PR TITLE
PHP Tutorial

### PR DESCRIPTION
**Pull request by valekjo**
https://docs.apify.com/tutorials/use-apify-from-php?version=feature/php-tutorial-experiment

Also check out the typeform link in the end.

### CODE REVIEW

**Review by m-murasovs — APPROVED** [review]
Great work! Please have one last scan to make sure I didn't change your meaning anywhere, and we're good to go ❤️

File: docs/tutorials/use_apify_from_php.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
 ---
 title: Use Apify from PHP
-description: Learn how to use Apify features via REST API from PHP using guzzle.
+description: Learn how to access Apify's REST API endpoints from you PHP using the guzzle package. Follow a tutorial to run an actor and download its data.
```
Comment (valekjo) [inline docs/tutorials/use_apify_from_php.md:3]:
```suggestion
description: Learn how to access Apify's REST API endpoints from PHP using the guzzle package. Follow a tutorial to run an actor and download its data.
```

File: docs/tutorials/use_apify_from_php.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 menuWeight: 3.93
 paths:
     - tutorials/use-apify-from-php
 ---
 
 # How to use Apify from PHP
 
-Apify's [RESTful API](https://docs.apify.com/api/v2#) allows you to use the platform from basically anywhere. Many projects are and will be built on PHP. This tutorial enables you to use Apify in these projects and it can be used in a variety of PHP frameworks.
+Apify's [RESTful API](https://docs.apify.com/api/v2#) allows you to use the platform from basically anywhere. Many projects are and will continue to be built using [PHP](https://www.php.net/). This tutorial enables you to use Apify in these projects in PHP and a variety of its frameworks.
```
Comment (valekjo) [inline docs/tutorials/use_apify_from_php.md:11]:
```suggestion
Apify's [RESTful API](https://docs.apify.com/api/v2#) allows you to use the platform from basically anywhere. Many projects are and will continue to be built using [PHP](https://www.php.net/). This tutorial enables you to use Apify in these projects in PHP and frameworks built on it.
```
Sounded strange, as those are not PHP's (own) frameworks.

File: docs/tutorials/use_apify_from_php.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
 
-Apify does not have an official client for PHP (yet), so we are going to use [guzzle](https://github.com/guzzle/guzzle), a great library for HTTP requests. We will cover a few endpoints that will show you the principles you can use for all Apify API endpoints.
+Apify does not have an official PHP client for (yet), so we are going to use [guzzle](https://github.com/guzzle/guzzle), a great library for HTTP requests. By covering a few fundamental endpoints, this tutorial will show you the principles you can use for all Apify API endpoints.
```
Comment (valekjo) [inline docs/tutorials/use_apify_from_php.md:13]:
```suggestion
Apify does not have an official PHP client (yet), so we are going to use [guzzle](https://github.com/guzzle/guzzle), a great library for HTTP requests. By covering a few fundamental endpoints, this tutorial will show you the principles you can use for all Apify API endpoints.
```

**Review by davidjohnbarton — APPROVED** [review]
All looks good to me!

**Pull request merged**